src/utils/inputValidation/preloadedData.js

```diff
@@ -0,0 +1,56 @@
+import { isObject, isString } from '../lang';
+import logFactory from '../logger';
+const log = logFactory('', {
+  displayAllErrors: true
```

Comment:
I'm not so sure about this... Why is it true here? What's the reasoning behind it?